webapp/src/widgets/valueSelector.tsx

```diff
@@ -54,6 +54,7 @@ const ValueSelectorLabel = React.memo((props: LabelProps): JSX.Element => {
                 <span className='Label-text'>{option.value}</span>
                 {onDeleteValue &&
                     <IconButton
+                        data-testid={`delete-value-${option.id}`}
```

Comment:
Probably you can use here the `.CloseIcon` class to test it, instead of the need of an specific.